endpoints/openrtb2/auction.go

```diff
@@ -276,6 +276,30 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request) (req *openrtb_
 		return
 	}
 
+	// Fetch Channel Information and Update Channel Object in config with this information
+	deps.cfg.Channel = openrtb_ext.ExtRequestPrebidChannel{}
+	channelInfo, err := getChannelInfoFromRequest(requestJson)
```

Comment:
You should use the `RequestWrapper` here instead. It's already in scope in this method.